core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -41,54 +38,43 @@
 public abstract class RetryingClient<I extends Request, O extends Response>
         extends SimpleDecoratingClient<I, O> {
 
-    private static final AttributeKey<Long> RESPONSE_TIMEOUT_DEADLINE_NANOS =
-            AttributeKey.valueOf("RESPONSE_TIMEOUT_DEADLINE_NANOS");
+    private static final AttributeKey<RetryingClientState> RETRYING_CLIENT_STATE =
+            AttributeKey.valueOf(RetryingClient.class, "RETRYING_CLIENT_STATE");
```

Comment:
`"STATE"` should be enough because `valueOf(Class, String)` will use the FQCN as a prefix.